app/scripts/directives/resourceServiceBindings.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+angular.module('openshiftConsole').component('resourceServiceBindings', {
+  controller: [
+    '$filter',
+    'DataService',
+    'BindingService',
+    'CatalogService',
+    ResourceServiceBindings
+  ],
+  controllerAs: '$ctrl',
+  bindings: {
+    project: '<',
+    projectContext: '<',
+    apiObject: '<',
+    createBinding: '&'
+  },
+  templateUrl: 'views/directives/resource-service-bindings.html'
+});
+
+
+function ResourceServiceBindings($filter, DataService, BindingService, CatalogService) {
+  var ctrl = this;
+  ctrl.bindings = [];
+  ctrl.bindableServiceInstances = [];
+  ctrl.serviceClasses = [];
+  ctrl.serviceInstances = [];
+  ctrl.secrets = [];
+  ctrl.showBindings = CatalogService.SERVICE_CATALOG_ENABLED;
```

Comment:
I think we should be checking for `ENABLE_TECH_PREVIEW_FEATURE.pod_presets` as well. If pos presets aren't enabled, we shouldn't try to get the bindings for a resource.

This might be a problem on the overview already as well.